aip/general/0131/1-guidance.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Maybe this isn't the place for this opinion, but that won't stop me from sharing it.

The change log has always felt unnecessary to me, and seeing it being added as a new yaml for each section exacerbates my feelings of it.

It seems like we should be able to make use of the GitHub API to autogenerate this instead of manually writing it out (says the guy not writing the code).

Reply:
@rhamiltonsf and I discussed this in a video call. TLDR:

- We have a changelog because it was requested by readers.
- We could maybe do this (determine from commits) going forward, but it would be very painful to do retroactively.
  - It likely would not stand up to massive fork and merge very well.